build/jest.transform.js

```diff
@@ -0,0 +1,4 @@
+var webpackConfig = require('./webpack.config.js')
+var babelConfig = webpackConfig.module.rules[0].use[0].query
```

Comment:
it seems like `babelConfig` should be defined in a separate module and required here and in the webpack config  :)